### ISSUE TITLE
Python is outdated

### ISSUE THREAD

**Issue by rugk** [fermiumlabs/latex-docker#19]
Python used for installing pygments is outdated, e.g. if you run (I just wanted to potentially update pygments):
```sh
$ pip install pygments
DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
Requirement already satisfied: pygments in /usr/local/lib/python2.7/dist-packages (2.5.2)
```

**Comment by ddavidebor** [comment]
Mmmh... this image needs indeed a little bit of love...

**Comment by ddavidebor** [comment]
Fixed in #22
